test/Microsoft.DocAsCode.MarkdownLite.Tests/GfmTest.cs

```diff
@@ -805,6 +805,24 @@ public void TestTable_Html_Div()
 
         [Fact]
         [Trait("Related", "Markdown")]
+        public void TestInlineHtml_Span()
+        {
+            // 1. Prepare data
+            var source = @"
+<span>a</span>
```

Comment:
Consider adding a `<span data-ttu-id="abcd-123">b</span>` case to test attributes in html tags.